### DISCUSSION TITLE
I would like to be able to copy the next authorization code even with a long tap.

### DISCUSSION THREAD

**Discussion by rarikuma** [ente-io/ente#1043] [Enhancements]
In the ente auth Android/iOS application, copying of the authentication code is executed with a "single tap" or a "long tap".
Also, a "single tap" on the display position of the next authentication code will execute the copying of the next authentication code.
However, with "long tap", the current authentication code is copied even if you tap the position where the next authentication code is displayed.
I would like to be able to copy the next authentication code with a "long tap" as well.

**Comment by vishnukvmd** [comment]
If you tap on the next code, it will copy that into your clip board.

Please reopen if this does not fix the issue!

**Comment by rarikuma** [comment]
At present, I understand that a single tap can copy the next chord, but when I first started using the application, I did not realize that the next chord could be copied because I was using a long tap to copy.
I feel uncomfortable with the asymmetry in behavior where copying the current code works with both a single tap and a long tap, but copying the next code only works with a single tap. This is why I made the request.
You could say it is a matter of personal preference, but I hope you will consider it.

**Reply by vishnukvmd** [reply]
Reopening this as a discussion so more folks can chime in if they feel the same way.
